cmd/tf_operator/app/server.go

```diff
@@ -61,6 +61,7 @@ func Run(opt *options.ServerOption) error {
 	glog.Infof("Go Version: %s", runtime.Version())
 	glog.Infof("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
 	if opt.PrintVersion {
+		fmt.Printf("%v\n", version.Version)
```

Comment:
Thanks for your PR, but I am not sure if it is a good solution. When you use the command `tf-operator -version -logtostderr`, It will print

```
I0203 16:49:35.991400    8213 server.go:73] tf_operator Version: 0.3.0+git
I0203 16:49:35.991406    8213 server.go:74] Git SHA: Not provided.
I0203 16:49:35.991410    8213 server.go:75] Go Version: go1.8.3
I0203 16:49:35.991412    8213 server.go:76] Go OS/Arch: linux/amd64
0.3.0+git
```

I prefer to set alsologtostderr to `true` by default to resue the glog to print version, but if you do not like it, please print all version info by fmt:

```go
fmt.Printf("tf_operator Version: %v", version.Version)
fmt.Printf("Git SHA: %s", version.GitSHA)
fmt.Printf("Go Version: %s", runtime.Version())
fmt.Printf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
```

Reply:
@gaocegege well, I think glog should be used to print _log_ for debugging; what -version option print is not log, so I think it's fine. :) 
And we should also correct other misusing part like:
https://github.com/tensorflow/k8s/blob/master/cmd/tf_operator/app/server.go#L94-L97,
we should use glog instead. what do you think?

Reply:
fmt.Errorf() formats according to a format specifier and returns the string as a value that satisfies error, and the function will not print any message. And glog.Errorf() logs to the ERROR, WARNING, and INFO logs. Arguments are handled in the manner of fmt.Printf; a newline is appended if missing.

I think they have different behaviour, as for here it is used to return an error so I think it should be  fmt 🤔 The error will be printed here: https://github.com/tensorflow/k8s/blob/master/cmd/tf_operator/main.go#L33.

Reply:
@gaocegege , my bad, you're right. so I think this pr should be fine. 😆

Reply:
I think version is not enough, maybe 

```go
fmt.Printf("tf_operator Version: %v", version.Version)
fmt.Printf("Git SHA: %s", version.GitSHA)
fmt.Printf("Go Version: %s", runtime.Version())
fmt.Printf("Go OS/Arch: %s/%s", runtime.GOOS, runtime.GOARCH)
```

is better 🤔  WDYT @caogj

Reply:
@gaocegege sure. like kubectl version:
```
$kubectl version
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.2", GitCommit:"bdaeafa71f6c7c04636251031f93464384d54963", GitTreeState:"clean", BuildDate:"2017-10-24T19:48:57Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.3+coreos.0", GitCommit:"f96beb2e925b0bb4e6ac0de746c7155dcd8fdc10", GitTreeState:"clean", BuildDate:"2017-11-13T10:18:37Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
```
1. remove unused function PrintVersion
2. add version.Info()
```
$./tf_operator -version -logtostderr
I0203 18:37:41.892546   16458 server.go:54] EnvKubeflowNamespace not set, use default namespace
I0203 18:37:41.892659   16458 server.go:58] Version: 0.3.0+git, Git SHA: Not provided., Go Version: go1.9.2, Go OS/Arch: darwin/amd64
Version: 0.3.0+git, Git SHA: Not provided., Go Version: go1.9.2, Go OS/Arch: darwin/amd64
```